### ISSUE TITLE
connect()

### ISSUE THREAD

**Issue by KaiTroendle** [studioimaginaire/phue#155]
If I want to connect to the Bridge form outside of the network. It gives me this error: PhueRequestTimeout: (None, 'GET Request to ip-adresse/api/RIuZMNixV0wjKi93PFOKd4XSNfjhQWjBsKbgqawR/lights/ timed out.')

Is there a way to connect with the bridge if you are not in the network

**Comment by redaxmedia** [comment]
I don't think this is possible for security reason accept you are using an VPN to your network or find a hacky way expose the Hue bridge to the internet but I guess this is a dead end.

From my point of view - your question is not related to this library overall.

**Comment by owendelong** [comment]
This isn't about the library... When you're off net, unless you have the IPv6 address of your bridge (assuming it has one) and you open up firewall rules to permit you to reach it via said IPv6 address (and your client has IPv6 support), you're not going to be able to reach the private IPv4 address of your bridge.

You can use "Out of Home" access through the myhue cloud in some cases, but not with phue.
